### PR TITLE
CI on Windows

### PR DESCRIPTION
**Pull request by Alexander-Makaryev**
Created a configuration for CI on Windows with Azure Pipeline.
Example of run can be seen on this branch.

### CODE REVIEW

**Comment by Alexander-Makaryev** [comment]
Last commit - it is not new fail, I just missed one while marked them.

File: hpat/tests/test_io.py [hunk 1]
```diff
@@ -1,4 +1,5 @@
 import unittest
+import sys
```
Comment (dmitrii-zagornyi) [inline hpat/tests/test_io.py]: Instead of sys.platform use platform.system(). This value is more predictable.

Comment (shssf) [reply hpat/tests/test_io.py]: @dmitrii-zagornyi Good catch!
"The output of sys.platform and os.name are determined at compile time. platform.system() determines the system type at run time."
https://stackoverflow.com/a/11674977/4428424

Comment (Alexander-Makaryev) [reply hpat/tests/test_io.py]: done

**Pull request merged**